boost.sh

```diff
@@ -1464,6 +1463,7 @@ printf "$format" "BOOST_SRC:" "$BOOST_SRC"
 printf "$format" "ANDROID_NDK_ROOT:" "$ANDROID_NDK_ROOT"
 printf "$format" "ANDROIDBUILDDIR:" "$ANDROIDBUILDDIR"
 printf "$format" "LINUXBUILDDIR:" "$LINUXBUILDDIR"
```

Comment:
Sqaush this into previous commit pls.

Reply:
I will be squashing the entire PR at merge time.